app/routes.py

```diff
@@ -1,40 +1,41 @@
 
-from flask import Blueprint, request, jsonify
+from flask import Blueprint, request
 import os
 from dotenv import load_dotenv
 import requests
 
 load_dotenv()
 
-proxy_bp = Blueprint("proxy_bp", __name__)
+bp = Blueprint("proxy_bp", __name__)
 
 location_key = os.environ.get("LOCATION_KEY")
 weather_key = os.environ.get("WEATHER_KEY")
 
-@proxy_bp.route("/location", methods=["GET"])
+@bp.get("/location")
 def get_lat_lon():
     loc_query = request.args.get("q")
     if not loc_query:
-        return {"message": "must provide q parameter (location)"}
+        return {"message": "must provide q parameter (location)"}, 400
 
     response = requests.get(
         "https://us1.locationiq.com/v1/search.php",
         params={"q": loc_query, "key": location_key, "format": "json"}
     )
 
-    return jsonify(response.json())
+    return response.json(), 200
 
-@proxy_bp.route("/weather", methods=["GET"])
+@bp.get("/weather")
 def get_weather():
     lat_query = request.args.get("lat")
     lon_query = request.args.get("lon")
     print(lat_query, lon_query)
+    print(weather_key)
```

Comment:
How do you feel about removing these? I feel a little weird about leaving print statements in our routes, but I don't feel super strongly if there's a reason to leave them in.

Reply:
Oh! I left the second one in there by accident (the key wasn't working but it just took a second to activate). I can remove it! The first one I am okay with removing as well. I wasn't quite sure why they were there, I figured they were for the students to check the coordinates, but that is something they can do themselves. I can remove it.